torch_xla/csrc/tensor_methods.cpp

```diff
@@ -1087,9 +1087,11 @@ XLATensorPtr XLATensor::exp(const XLATensorPtr& input) {
 XLATensorPtr XLATensor::expand(const XLATensorPtr& input,
                                std::vector<int64_t> size) {
   auto input_shape = input->shape();
```

Comment:
I am a bit hesitant because we actually did not lower the expand as view(and yes this is a bug). I felt like in this case we should not trying to make `is_alias` works for expand tensor.

Reply:
For view ops with operands that have storages, PyTorch asserts the results should be alias of the inputs. Here is the assertion: https://paste.googleplex.com/4898491405959168. Nick tried once making expand as a proper view op and failed. I guess it's probably hard to implement. Ultimately we can leverage functionalization to properly fix this kinds of view problems all at once. I guess we can just patch the assertion and wait until LTC migration fully completes, and leave a small window for potential bugs.

Do you anticipate any issues we might have?